mapper/orm-coordination-database-polling/src/main/java/org/hibernate/search/mapper/orm/coordination/databasepolling/event/impl/DatabasePollingOutboxEventAdditionalJaxbMappingProducer.java

```diff
@@ -54,6 +54,7 @@
 			"        <property name=\"entityIdHash\" type=\"integer\" index=\"entityIdHash\" />\n" +
 			"        <property name=\"payload\" type=\"materialized_blob\" />\n" +
 			"        <property name=\"retries\" type=\"integer\" />\n" +
+			"        <property name=\"processAfter\" type=\"Instant\" />\n" +
```

Comment:
Any reason not to add a database index on this property, as requested [here](https://hibernate.atlassian.net/browse/HSEARCH-4194?focusedCommentId=108586)?

If we're going to run queries against this table every 50 milliseconds, we might as well optimize them.